Gopkg.lock

```diff
@@ -191,11 +191,11 @@
 
 [[projects]]
   branch = "master"
-  digest = "1:3d4a946cf7d13080ff1e15530bf53b7607fe7bbe5edf1c045b637b96526f463d"
+  digest = "1:a19155ec7100c8a8583c699403907c61609464b09b874ede9c178449878bde1e"
   name = "github.com/google/go-github"
```

Comment:
can we switch to a semver dependency for this package? at the moment it has `master` in the toml